content/en/dashboards/ddsql_editor/reference/statements.md

```diff
@@ -28,18 +31,24 @@ SELECT [ ALL | DISTINCT ] select_expr, ...
 
 `select_expr`
 : Any expression that returns a value. It may be a constant, function call, aggregate, window, or the special expression `*`. This is the part of the query that specifies the output of the SELECT statement, and in relational algebra it is known as the projection.
- 
+
+`message_pattern`
+: A textual pattern for [full-text search][3], where available.
+
+`index_name`
+: An identifier for a [logs index][4].
+
 `rel_source`
-: A correlation (a table name or alias) or a parenthesized [DQL expression][3].
+: A correlation (a table name or alias) or a parenthesized [DQL expression][5].
 
-`JOIN_TYPE`
+`join_type`
```

Comment:
Changed to lower-case, as this is a placeholder, not a keyword. The syntax above uses all-caps for keywords and lower-case for placeholders, so I'm aligning with that.